docs/Installation-Windows.md

```diff
@@ -126,19 +126,28 @@ git clone https://github.com/Unity-Technologies/ml-agents.git
 If you don't want to use Git, you can always directly download all the files
 [here](https://github.com/Unity-Technologies/ml-agents/archive/master.zip).
 
+The `UnitySDK` subdirectory contains the Unity Assets to add to your projects.
+It also contains many [example environments](Learning-Environment-Examples.md)
+that can be used to help get you familiar with Unity.
```

Comment:
"that can be used to help get you familiar with Unity." --> "to help you get started."

Reply:
Fixed, I also fixed the exact same wording in the Installation.md doc.